clientbuffer.cpp

```diff
@@ -52,9 +52,14 @@ class CClientBufferMod : public CModule
     MODCONSTRUCTOR(CClientBufferMod)
     {
         AddHelpCommand();
-        AddCommand("AddClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnAddClientCommand), "<identifier>", "Add a client.");
-        AddCommand("DelClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnDelClientCommand), "<identifier>", "Delete a client.");
-        AddCommand("ListClients", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnListClientsCommand), "", "List known clients.");
+        AddCommand("AddClient", t_d("<identifier> [timelimit]"), t_d("Add a client with optiona time limit."),
```

Comment:
This module is not localized. Making strings go through the localization system is a pointless overhead unless you are also adding localization. Also, such changes need to be in their own commit / pull request.

Reply:
As mentioned previously, if more per-client parameters were to be added, they would have to be all specified one after the other, without the possibility to skip any.

Specifying settings here is also superfluous to `ChangeClient` / `SetClientTimeLimit`, so I suggest not changing the semantics of the `AddClient` command.